src/ZODB/ExportImport.py

```diff
@@ -31,7 +31,7 @@
 
 class ExportImport(object):
 
-    def exportFile(self, oid, f=None):
+    def exportFile(self, oid, f=None, bufsize=1 << 16):
```

Comment:
I would find `64*1024` more readable than `1 << 16`.

Reply:
Funny, I find 1 << 16 more readable.  It's practically a literal to me.